content/general/concepts/unit-testing/unit-testing.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Unit Testing'
+Description: 'Unit testing involves testing ] parts of code to verify its correctness.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Debugging'
+  - 'Development'
+CatalogContent:
+  - 'learn-javascript-unit-testing'
+---
+
+Unit testing is an essential part of the coding process to verify all parts of code work correctly.
+
+A part or unit of code can be anything from a class function to a small snippet of code that performs a specific task. By testing individual components, it makes it easier to find exactly where an error lies.
+
+There are usually two types of unit testing — manual and automated testing.
+
+## Manual Testing
+
+Manual testing is when tests are being done manually by people. It can involve clicking on objects, providing input, or scrolling through items.
```

Comment:
```suggestion
Manual testing is when tests are done directly by people. It can involve clicking on objects, providing input, or scrolling through items.
```